### DISCUSSION TITLE
纯小白，挂载阿里云盘这样了，求助各位大佬

### DISCUSSION THREAD

**Discussion by yybcx** [AlistGo/alist#6344] [Q&A]
![20240413_160109](https://github.com/alist-org/alist/assets/144800692/b9ca27d5-c712-49e5-b5aa-679d32445bc0)
是哪里不对吗

**Comment by xuanxuan1231** [comment]
你的网络问题

**Reply by ddns6** [reply]
openapi。编辑检查替换api，api套cf了吗？
